### ISSUE TITLE
rice seeding date   CO_320:0000514 

### ISSUE THREAD

**Issue by cooperl09** [Planteome/CO_320-rice-traits#41]
This is not a trait 
https://browser.planteome.org/amigo/term/CO_320:0000514

**Comment by marieALaporte** [comment]
That's why it is not mapped to a TO trait. How should we deal with those?
This is an important parameter that people record and that is used in different analysis such as crop models, ....

**Comment by cooperl09** [comment]
Non-trait terms have been removed from the mapping file, now that they have all been moved to the new trait-mappings repo.
